.github/workflows/waspc-ci.yaml

```diff
@@ -54,6 +54,21 @@ jobs:
           - ubuntu-20.04
           - macos-latest
           - windows-latest
+        node-version:
+          - 'latest'
```

Comment:
@Martinsos Should we run our "usual" tests on Node.js 18 or latest Node.js?

We now have the following matrix:
```
(Win, latest)
(MacOS, latest)
(Ubuntu, latest)
(Ubuntu, 18)
(Ubuntu, 20)
```

Reply:
What do you mean by usual tests?

I would say this matrix is good! It covers everything.

Aha I see what you mean: should we have MacOS + latest, or MacOS + 18?

Ha good question. I don't think it is super important. I would probably go with MacOS + 18/lowest, because if it works for lowest, it should also work for the higher versions (although we have seen that is not always true, with the polifyll thing we had).
But combo of Ubuntu + node versions should really cover it all, which is why I think this choice is not so crucial.

Reply:
Okay, let's keep it like this then :)